src/ovs/aca_vlan_manager.cpp

```diff
@@ -207,12 +214,25 @@ int ACA_Vlan_Manager::create_l2_neighbor(string virtual_ip, string virtual_mac,
   stArpCfg.ipv4_address = virtual_ip;
   stArpCfg.vlan_id = internal_vlan_id;
 
+  // auto overall_rc_ptr = & overall_rc;
+
+  // auto arp_config_ptr = & stArpCfg;
+
   overall_rc = ACA_ARP_Responder::get_instance().create_or_update_arp_entry(&stArpCfg);
 
-  ACA_LOG_DEBUG("create_l2_neighbor arp entry with ip = %s, vlan id = %u and mac = %s\n",
-                virtual_ip.c_str(), internal_vlan_id, virtual_mac.c_str());
+  // overall_rc = ACA_ARP_Responder::get_instance().create_or_update_arp_entry(&stArpCfg);
+  // marl::schedule([=] {
+  // });
+
+  // ACA_LOG_DEBUG("create_l2_neighbor arp entry with ip = %s, vlan id = %u and mac = %s\n",
+  //               virtual_ip.c_str(), internal_vlan_id, virtual_mac.c_str());
 
-  ACA_LOG_DEBUG("%s", "ACA_Vlan_Manager::create_l2_neighbor <--- Exiting\n");
+  // ACA_LOG_DEBUG("%s", "ACA_Vlan_Manager::create_l2_neighbor <--- Exiting\n");
+  processed_neighbor_state_count ++;
+  if(processed_neighbor_state_count.load() == 970000){ // send barrier request at 970k th neighbor
```

Comment:
This is for testing only, please revert to production code

Reply:
Removed.